### PR TITLE
[zh]update  content/zh/boilerplates/trace-generation.md

### PR DESCRIPTION
**Pull request by yanrongshi**
Please provide a description for what this PR is for.

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

- [ ] Configuration Infrastructure
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Policies and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

File: content/zh/boilerplates/trace-generation.md [hunk 1]
```diff
@@ -1,9 +1,8 @@
 ---
 ---
-要查看追踪数据，必须向服务发送请求。请求的数量取决于 Istio 的采样率。
-采样率在安装 Istio 时设置，默认采样速率为 1%。在第一个跟踪可见之前，您需要发送至少 100 个请求。
+要查看追踪数据，必须向服务发送请求。请求的数量取决于 Istio 的采样率，可以使用 [Telemetry API](/docs/tasks/observability/telemetry/)进行配置。采样率在安装 Istio 时设置，默认采样速率为 1%。在第一个跟踪可见之前，您需要发送至少 100 个请求。
```
Comment (yanrongshi) [inline content/zh/boilerplates/trace-generation.md:3]:
```suggestion
要查看追踪数据，必须向服务发送请求。请求的数量取决于 Istio 的采样率，可以使用 [Telemetry API](/zh/docs/tasks/observability/telemetry/)进行配置。采样率在安装 Istio 时设置，默认采样速率为 1%。在第一个跟踪可见之前，您需要发送至少 100 个请求。
```

File: content/zh/boilerplates/trace-generation.md [hunk 2]
```diff
@@ -1,9 +1,8 @@
 ---
 ---
-要查看追踪数据，必须向服务发送请求。请求的数量取决于 Istio 的采样率。
-采样率在安装 Istio 时设置，默认采样速率为 1%。在第一个跟踪可见之前，您需要发送至少 100 个请求。
+要查看追踪数据，必须向服务发送请求。请求的数量取决于 Istio 的采样率，可以使用 [Telemetry API](/zh/docs/tasks/observability/telemetry/)进行配置。采样率在安装 Istio 时设置，默认采样速率为 1%。在第一个跟踪可见之前，您需要发送至少 100 个请求。
```
Comment (litong01) [inline content/zh/boilerplates/trace-generation.md:3]: I think it maybe better like the following:

请求的数量取决于 Istio 的采样率，采样率可以使用 [Telemetry API](/zh/docs/tasks/observability/telemetry/)进行配置. Istio的默认采样速率为 1%, 如果默认采样率没有改变的话，想要观察到第一个追踪数据，就需要至少发送100个请求。

File: content/zh/boilerplates/trace-generation.md [hunk 3]
```diff
@@ -1,9 +1,8 @@
...
 使用以下命令向 `productpage` 服务发送 100 个请求：
 
 {{< text bash >}}
-$ for i in `seq 1 100`; do curl -s -o /dev/null http://$GATEWAY_URL/productpage; done
+$ for i in $(seq 1 100); do curl -s -o /dev/null "http://$GATEWAY_URL/productpage"; done
```
Comment (litong01) [inline content/zh/boilerplates/trace-generation.md:7]: does this need to be changed?

**Review by litong01 — CHANGES_REQUESTED** [review]

File: content/zh/boilerplates/trace-generation.md (hunk 2 above)
Comment (yanrongshi) [reply content/zh/boilerplates/trace-generation.md:3]: Thanks

**Comment by zirain** [comment]
this doc is actually unused
BTW, the title of this PR is wrong.

File: content/zh/boilerplates/trace-generation.md [hunk 4]
```diff
@@ -1,7 +1,6 @@
 ---
 ---
-要查看追踪数据，必须向服务发送请求。请求的数量取决于 Istio 的采样率。
-采样率在安装 Istio 时设置，默认采样速率为 1%。在第一个跟踪可见之前，您需要发送至少 100 个请求。
+要查看追踪数据，必须向服务发送请求。请求的数量取决于 Istio 的采样率，可以使用 [Telemetry API](/zh/docs/tasks/observability/telemetry/)进行配置。采样率在安装 Istio 时设置，默认采样速率为 1%。在第一个跟踪可见之前，您需要发送至少 100 个请求。
```
Comment (zirain) [inline content/zh/boilerplates/trace-generation.md:3]: 默认采样速率为 1%。在第一个跟踪可见之前，您需要发送至少 100 个请求。
```suggestion
要查看追踪数据，必须向服务发送请求。请求的数量取决于 Istio 的采样率，可以使用 [Telemetry API](/zh/docs/tasks/observability/telemetry/)进行配置。默认采样速率为 1%。在第一个跟踪可见之前，您需要发送至少 100 个请求。
```

**Comment by zirain** [comment]
cc @litong01

**Comment by zirain** [comment]
@yanrongshi 最近一个提交需要revert下处理下CLA

**Comment by zirain** [comment]
@yanrongshi 你需要revert下commit，其中有个commit有我另外一个未签署cla的邮箱

命令大概如下：
```console
git reset HEAD~5
git add .
git commit -m "update content/zh/boilerplates/trace-generation.md"
git push -f
```

**Comment by windsonsea** [comment]
Hope you can check more. 
FYI: Giving  a newline char is much friendly to review.

**Comment by windsonsea** [comment]
Because of no response for a long time and such a small PR with less value, this PR is closed.

**Pull request closed without merging**